collections/_pages/features.html

```diff
@@ -81,32 +83,38 @@
   - question: Questions?
     answer: Contact us for any further questions at <a href="#">community@layer5.io</a>. -->
 ---
+
 <div class="features">
-    
 	{% for feature in page.features_table %}
-		<ul class="feature" style="display:flex;flex-direction:column;">
-			<li style="background: {{ feature.color }}">
-				<h3>{{ feature.name }}</h3>
-			</li>
-			{% for item in feature.features %}
-				<li {% if item.highlight %} class="highlighted"{% endif %}>{{ item.text }}</li>
-			{% endfor %}
-
-			{% if feature.call_to_action %}
-				<li class="features-cta" style="margin-top: auto;"><div class="features-cta start"><a href="{{ feature.call_to_action.link }}">{{ feature.call_to_action.text }} &rarr;</a></div></li>
-			{% endif %}
-		</ul>
+	<ul class="feature" style="display: flex; flex-direction: column">
```

Comment:
Is there any change here other than line wrapping?

Reply:
No, Only line wrapping.

Reply:
Gotcha. 👍